other/svg-icon-template.txt

```diff
@@ -46,7 +46,7 @@ export function Icon({
 }) {
 	if (children) {
 		return (
-			<span className={`inline-flex ${childrenSizeClassName[size]}`}>
+			<span className={`inline-flex items-center ${childrenSizeClassName[size]}`}>
```

Comment:
Adjusted gap size to 1.5 instead of the 1 currently.

Added items-center as the icon and text are not vertically centered. 

Here is the how a LARGE sized delete button looks like without items-center  
![image](https://github.com/epicweb-dev/epic-stack/assets/143390/edc50038-f7f8-4cc2-98ba-cb806b8b80a8)

With items-center, it's better
![image](https://github.com/epicweb-dev/epic-stack/assets/143390/16d3e3fd-e293-44ca-b9ab-12d503b7ef84)